superset/databases/api.py

```diff
@@ -280,6 +327,12 @@ def post(self) -> FlaskResponse:
             if new_model.driver:
                 item["driver"] = new_model.driver
 
+            # Return SSH Tunnel and hide passwords if any
+            if item.get("ssh_tunnel"):
+                item["ssh_tunnel"] = new_model.ssh_tunnel  # pylint: disable=no-member
```

Comment:
create a util function that does the popping/deletion and returns the object without those values

Reply:
Sure! I'll indeed use our `mask` as I did here: https://github.com/apache/superset/pull/22513/files#diff-f9deb9b63f5f2db38b811ff092cf90b0d4d558223f743bbbb5b203c203d2d590R336 but sure, let me add that util. Thanks!